pkg/spec3/fuzz.go

```diff
@@ -0,0 +1,250 @@
+package spec3
+
+import (
+	fuzz "github.com/google/gofuzz"
+	"github.com/lucasjones/reggen"
```

Comment:
Can we add this dependency? That seems unfortunate. Any other way to generate valid arbitrary strings?

Reply:
It's a single go file https://github.com/lucasjones/reggen/blob/master/reggen.go, if it's not a desired dependency, I can write a custom function to generate valid arbitrary strings.